Assets/Scripts/Visuals/UiBuilder/AssetPropertyAtom.cs

```diff
@@ -8,8 +8,18 @@ public class AssetPropertyAtom : Atom
         public new class Data : Atom.Data
         {
             public string propertyName;
-            public AssetMetadata assetMetadata;
-            public Action<Guid> onClick;
+
+            [NotNull]
+            public SetValueStrategy<Guid> valueBindStrategy;
```

Comment:
I have seen the following 4 fields on several classes. Is there a way to reduce redundancy?